python/instrumentation/openinference-instrumentation-bedrock/tests/test_instrumentor.py

```diff
@@ -271,6 +271,208 @@ def test_invoke_client_with_missing_tokens(
         )
     assert attributes == {}
 
+@pytest.mark.parametrize("use_context_attributes", [False, True])
```

Comment:
a test for multiple messages would be good although that seems like it's mayb ecomign later?